apps/src/templates/Button.jsx

```diff
@@ -303,6 +313,18 @@ const styles = {
       paddingRight: 10,
       lineHeight: '40px'
     }
+  },
+  textButton: {
+    color: color.teal,
+    border: 'none',
+    backgroundColor: 'unset',
```

Comment:
unset isn't supported in IE ([MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/unset)), which should be fine if we're deprecating IE support, but just calling it out to make sure